docs/wearables/providers/guides/my_fitness_pal.mdx

```diff
@@ -4,7 +4,23 @@ title: "MyFitnessPal (Beta)"
 
 ## Connecting MyFitnessPal to Vital
 
-You can now link MyFitnessPal accounts to Vital! You can do this with Vital Link Widget or programatically like other [password](/api-reference/link/link-email-provider) providers.
+You can now link MyFitnessPal accounts to Vital! You can do this with Vital Link Widget or programatically like other [password](/api-reference/link/link-password-provider) providers.
+
+## Before Connecting
+
+You can share your MyFitnessPal diary with Vital by locking your diary with key.
```

Comment:
Can we describe this a bit better

Reply:
You resolved this, are you sure you are happy with the wording?